core/ulog/log_test.go

```diff
@@ -134,6 +134,24 @@ func TestConfiguredLoggerWithTextFormatter(t *testing.T) {
 	})
 }
 
+func TestConfiguredLoggerWithTextFormatter_NonDev(t *testing.T) {
+	dev := development
+	development = false
+	defer func() {
+		development = dev
+	}()
+	withLogger(t, func(tmpDir string, logFile string) {
+		log := Logger()
+		txt := true
+		log.Configure(Configuration{
+			Level:         "debug",
+			TextFormatter: &txt,
+		})
+		zaplogger := log.RawLogger()
+		assert.Equal(t, zap.DebugLevel, zaplogger.Level())
```

Comment:
Yes, you are right. My goal was to simplify it a bit, but its clearly not happening with the current state.  I am refactoring this code instead, and moving to Builder pattern. Its more clean for user access and testing as well.